velox/docs/ext/spark.py

```diff
@@ -0,0 +1,1113 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The Spark domain."""
```

Comment:
Is this a copy-paste from somewhere + minor tweaks? If so, let's add a comment to explain where this code came from.

@majetideepak Deepak, would you help review this code?

Reply:
Yes, I copy python.py [here](https://github.com/sphinx-doc/sphinx/blob/a13cf2c24dd16b37670ee1d359f511cbdfa4402d/sphinx/domains/python.py) and removed some unnecessary features for reducing the amount of code.